### PR TITLE
Add basic methods to handle objects in NeoFS

### LINKED ISSUE #850: NeoFS SDK: Create beta version implementation for objects
This includes the following NeoFS methods:

- Create object (without session)
- Read object (without session)
- Delete object

Tasks:
- [x] createObject
- [x] readObject
- [x] deleteObject

### LINKED ISSUE #872: NeoFS SDK: Convert neow3j NeoFS support architecture to figure

### LINKED ISSUE #909: NeoFS SDK: Add method to delete a client

### LINKED ISSUE #911: NeoFS SDK: aarch64 should be supported
NeoFS Loader should support `aarch64` as architecture, analogous to `arm64`.

image

More specifically, [this method](https://github.com/neow3j/neow3j/blob/253ce68b370903f99978c940178dc3c8d0612531/neofs/src/main/java/io/neow3j/neofs/lib/NeoFSLibUtils.java#L33) should be changed/adapted, maybe [in this line](https://github.com/neow3j/neow3j/blob/253ce68b370903f99978c940178dc3c8d0612531/neofs/src/main/java/io/neow3j/neofs/lib/NeoFSLibUtils.java#L35).

### PR DESCRIPTION
**Pull request by mialbu**
Closes #850
Closes #872 
Closes #909
Closes #911

### CODE REVIEW

File: int-tests/src/test-integration/java/io/neow3j/neofs/ObjectIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    public void testCreateAndDeleteObject() throws InterruptedException, InvalidProtocolBufferException {
+        String testText = "Neow3j is awesome!";
+
+        String objectId = neofsClient.createObject(containerId, testText.getBytes(UTF_8), account);
+        Thread.sleep(2000); // Wait for object creation to persist.
+        byte[] readBytes = neofsClient.readObject(containerId, objectId, account);
+        assertThat(new String(readBytes), is(testText));
+
+        String tombstoneId = neofsClient.deleteObject(containerId, objectId, account);
+        Thread.sleep(2000); // Wait for object deletion to persist.
```
Comment (gsmachado) [inline int-tests/src/test-integration/java/io/neow3j/neofs/ObjectIntegrationTest.java:81]: OHHH, can we change this somehow?

Rule number 1 about integration tests: never use `Thread.sleep()`.
Rule number 2 about integration tests: always follow rule number 1.

Comment (gsmachado) [reply int-tests/src/test-integration/java/io/neow3j/neofs/ObjectIntegrationTest.java:81]: Can we make a utils method to keep polling and checking if the object really got deleted?

File: neofs/src/main/java/io/neow3j/neofs/sdk/NeoFSClient.java [hunk 2]
```diff
@@ -77,6 +88,14 @@ private static String createClient(NeoFSLib lib, Account account, String neofsEn
...
+        return Numeric.toHexStringNoPrefix(ecKeyPair.getPrivateKey().getBytes());
+    }
+
+    public static String getPrivateKeyForNativeLib(Account account) {
+        return getPrivateKeyForNativeLib(account.getECKeyPair());
+    }
+
     //region client
 
     // Todo: Delete a client
```
Comment (gsmachado) [inline neofs/src/main/java/io/neow3j/neofs/sdk/NeoFSClient.java:101]: the delete client part will not be done? Like, disposal of a client?

If not, can you create an issue about that?

Comment (mialbu) [reply neofs/src/main/java/io/neow3j/neofs/sdk/NeoFSClient.java:103]: Yes, I want to include that. I'll open an issue about it.

Comment (mialbu) [reply neofs/src/main/java/io/neow3j/neofs/sdk/NeoFSClient.java]: This has been done in [09d3cb9](https://github.com/neow3j/neow3j/pull/908/commits/09d3cb97a954a7f3c16a7f0dea70e1e818babc85).

File: int-tests/src/test-integration/java/io/neow3j/neofs/ObjectIntegrationTest.java (hunk 1 above)
Comment (mialbu) [reply int-tests/src/test-integration/java/io/neow3j/neofs/ObjectIntegrationTest.java:81]: This issue is tracked in #910.

**Review by gsmachado — APPROVED** [review]
Looks good, @mialbu! 👍

**Pull request merged**